doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-comparison-legacy-new-ref.adoc

```diff
@@ -34,73 +34,77 @@ a|
 
 * For more information about creating rule and DMN-based test scenarios, see xref:test-designer-create-test-scenario-template-con[].
 * For more information about running the test scenarios, see xref:test-designer-run-test-proc[].
-* For more information about creating and running test scenarios(legacy), see xref:test-scenarios-legacy-create-proc[].
+* For more information about creating and running test scenarios (legacy), see xref:test-scenarios-legacy-create-proc[].
 
 a|*Adding GIVEN facts in test scenarios*
 
-You can insert and verify the *GIVEN* facts for the test.
+* You can insert and verify the *GIVEN* facts for the test.
 
 |`+`
 |`+`
 
 a|
 
-* For more information about adding GIVEN facts in new test scenarios designer, see xref:test-designer-create-test-scenario-template-con[].
-* For more information about adding GIVEN facts in test scenarios(legacy), see xref:test-scenarios-legacy-GIVEN-proc[].
+* For more information about adding *GIVEN* facts in new test scenario designer, see xref:test-designer-create-test-scenario-template-con[].
+* For more information about adding *GIVEN* facts in test scenarios (legacy), see xref:test-scenarios-legacy-GIVEN-proc[].
 
 a|*Adding EXPECT results in test scenarios*
 
-* It represents the objects and their fields whose exact values are checked based on the provided information.
 * The *EXPECT* section defines the expected results based on the *GIVEN* input facts.
+* It represents the objects and their fields whose exact values are checked based on the provided information.
 
 |`+`
 |`+`
 
 a|
 
-* For more information about adding EXPECT results in new test scenarios designer, see xref:test-designer-create-test-scenario-template-con[].
-* For more information about adding EXPECT results in test scenarios(legacy), see xref:test-scenarios-legacy-EXPECT-proc[].
+* For more information about adding *EXPECT* results in new test scenario designer, see xref:test-designer-create-test-scenario-template-con[].
+* For more information about adding *EXPECT* results in test scenarios (legacy), see xref:test-scenarios-legacy-EXPECT-proc[].
 
-|*KIE session*
+a|*KIE session*
 
-KIE session can be set on test scenario level settings.
+* You can set KIE session on test scenario level settings.
 
 |`+`
 |`+`
-|-
+|NA
 
 a|*KIE base on test scenario level*
 
-You can set KIE base on test scenario level settings.
+* You can set KIE base on test scenario level settings.
+
 |`-`
 |`+`
-|-
+|NA
 
 a|*KIE base on project level*
 
 You can set KIE base on project level settings.
 |`+`
 |`+`
-|-
+|NA
+
+a|*Simulated date and time*
+
+* You can set a simulated date and time for the legacy test scenario designer. 
 
-|*Simulated date and time*
 |`-`
 |`+`
-|-
+|NA
 
 a|*Rule flow group*
 
-You can specify a rule flow group to be activated so that all rules within that group can be tested.
+* You can specify a rule flow group to be activated to test all the rules in within that group.
```

Comment:
Suggested rewrite:

You can specify a rule flow group to be activated to test all the rules within that group. 

//removing in